lib/data.ts

```
@@ -41,9 +48,7 @@ export async function getRecentListings(): Promise<DisplayListingItem[]> {
 
   if (error) {
     console.error("Error fetching recent listings:", error.message);
-    Toast({
-      title: "Error fetching recent listings... Please refresh the page.",
-    });
+    throw new Error("Failed to fetch recent listings");
     return [];
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)